### PR TITLE
Личный проект: модуляция

### PR DESCRIPTION
**Pull request by akzhar**
---
:mortar_board: [Личный проект: модуляция](https://up.htmlacademy.ru/javascript/17/user/490939/tasks/12)

:boom: https://htmlacademy-javascript.github.io/490939-keksobooking-17/6/

### CODE REVIEW

File: js/apartments.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    var pinImg = pin.querySelector('img');
+    var pinX = apartment.location.x - dependencies.data.PIN_WIDTH / 2;
+    var pinY = apartment.location.y - dependencies.data.PIN_HEIGHT;
+    pin.style.left = pinX + 'px';
+    pin.style.top = pinY + 'px';
+    pinImg.src = apartment.author.avatar;
+    pinImg.alt = 'заголовок объявления'; // to be filled in the future
+    return pin;
+  }
+  function getFragmentWithPins(apartments) {
```
Comment (rkhaslarov) [inline js/apartments.js]: try to put space between functions =)

File: js/apartments.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+    return fragment;
+  }
+  window.apartments = {
+    generateApartments: function () {
+      var apartments = createApartments(dependencies.data.NUMBER_OF_OFFERS);
+      var fragment = getFragmentWithPins(apartments);
+      var mapPins = mapBlock.querySelector('.map__pins');
+      mapPins.appendChild(fragment);
+    }
```
Comment (rkhaslarov) [inline js/apartments.js]: for making it a bit smaller, you could create the function above and then just assign to the property by name like
{
generateApartments: generateApartments
}

Comment (rkhaslarov) [reply js/apartments.js]: it will make the export readable

File: js/apartments.js (hunk 1 above)
Comment (akzhar) [reply js/apartments.js]: Ok!

File: js/apartments.js (hunk 2 above)
Comment (akzhar) [reply js/apartments.js]: Good idea!

**Pull request merged**